### DISCUSSION TITLE
Command line option to override displayed hostname?

### DISCUSSION THREAD

**Discussion by rraszews** [vicwomg/pikaraoke#232] [Ideas]
I notice that the splash screen shows a URL of http://(IP address):5000 I know that's perfect for most people, but in my case, the pikaraoke box is on my private network, and I'd rather not give my guests direct access to it. I can set up NAT forwarding to allow them in through the firewall, but the URL they would use would be based on my public IP (or really, my public DNS name) rather than the IP of the pikaraoke box itself. It would be nice if there was a way to force the splash screen to show a URL and generate a QR code based on command-line input so I could tell it the public-facing URL.

**Comment by bingobob** [comment]
Yes please
